src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 500;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+body {
+  margin: 20px 8px 8px;
+}
+
+.search {
+  font-family: Avenir, sans-serif;
```

Comment:
move font-family to `html` selector and set `inherit`
you use it many times

Reply:
In the task autor wrote "use `@font-face` for fonts" .  I can do that. It will be ok?
body {
  margin: 20px 8px 8px;
  font-family: Avenir, sans-serif;
  font-weight: 500;
  --color-placeholder: #3d4e61;
  --color-box-shadow: rgba(61, 78, 97, 0.2);
}